manuscript/template.tex

```diff
@@ -0,0 +1,667 @@
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Modelling Language and Methodology}
+The languages considered were SysML V1 \cite{sys_ml}, SysML V2 \cite{sys_ml2}, PlantUML \cite{plantuml}, and the Mach 30 modelling language (M30ML) \cite{mach30_git} with the flight software team conducting a trade study to determine the most suitable.  The down selection criteria that the SeaLion CubeSat flight software team has taken into consideration is shown in table \ref{tab:language}.
+
+\begin{table}[H] 
+	\caption{Modeling Language Downselect}
+	\label{tab:language}
+	\newcolumntype{C}{>{\centering\arraybackslash}X}
+	\begin{tabularx}{\textwidth}{CCCCC}
+	\toprule
+	\textbf{Criteria}  & \textbf{SysML v1}  & \textbf{SysML v2}  & \textbf{PlantUML}  & \textbf{M30ML}\\
+	\midrule
+	Extensible ontology language                       & X        & X        & X        & X     \\ \hline
+	Supports both textual \& graphical view generation &          & X        &          & X     \\ \hline
+	Lightweight textual syntax                         &          & X        & X        & X     \\ \hline
+	Relatively minimal overhead with modern doctools   &          &          & X        & X     \\ \hline
+	Supports execution semantics                       &          & X        &          &       \\
+	\bottomrule
+	\end{tabularx}
+\end{table}
+
+M30ML was chosen for its lightweight human and machine-readable textual syntax, file-based model interchange support (for persisting models directly on the local filesystem), ability to generate both textual and graphical views, and relatively minimal overhead with modern doctools \cite{mach30_git}.  The lightweight textual syntax and minimal over head is especially important for for a team that had very minimal experience working with such tools.  Other candidate modelling languages lacked in many regards compared to M30ML in these criteria and thus, M30ML was selected.  SysML v2 had a good number of characteristics that M30ML had, however, the lack of minimal overhead with modern doctools prevented its adoption.  
+
+\subsection{Ontological Modeling Language}
+M30ML was developed using the Ontological Modeling Language (OML) as its basis.  OML is a language that enables defining systems engineering vocabularies and using them to describe systems \cite{oml_language}.  OML, inspired by Web Ontology Language 2 (OWL2) and the Semantic Web Rule Language (SWRL), is meant to be a more gentler and more disciplined method of aforementioned standard for use in systems engineering \cite{oml_language}.  OWL2 does not conform easily to individual modelling rules without tooling support; thus, OML was created.  OML is a tool to improve the speed of modeling and the quality of models while in a more concise and human-friendly high-level external representation \cite{oml_origin_and_rationale}. 
```

Comment:
m30ml is no longer being developed with OML, in favor of LinkML for its lightweight YAML-based syntax and lightweight toolchain

Reply:
Resolved bbd10cadce0dc6665d7e685b218b57c669172dc5